packages/expect/src/matchers.js

```diff
@@ -87,7 +87,16 @@ const matchers: MatchersObject = {
   toBeCloseTo(actual: number, expected: number, precision?: number = 2) {
     const secondArgument = arguments.length === 3 ? 'precision' : null;
     ensureNumbers(actual, expected, '.toBeCloseTo');
-    const pass = Math.abs(expected - actual) < Math.pow(10, -precision) / 2;
+
+    let pass = false;
+    //If both actual and expected are equal to the Infinity global
```

Comment:
Let's just remove the comments, they just say what the code says (again, sorry for the run around)

Reply:
No problem, I agree with you, I think the code is pretty self-explanatory, but was afraid I was the only one so I commented. Removing them now